packages/button/src/vaadin-button-base.js

```diff
@@ -93,6 +93,10 @@ export const buttonStyles = css`
         --vaadin-icon-color: Canvas;
       }
 
+      ::slotted(*) {
+        forced-color-adjust: auto;
```

Comment:
Is this safe in terms of specificity? It would override `forced-color-adjust: none` on a child element, which it might have.

![image](https://github.com/user-attachments/assets/653101f3-4718-43e1-9d1b-f864451eeb59)

Reply:
I’m not entirely sure about this. Perhaps it would be better to style all buttons the same in forced color mode, and not treat primary buttons any differently. That’s the reason why I use `forced-color-adjust: none` for them, so that i can invert the colors on primary buttons. 

In terms of specificity, I think if a component defines `forced-color-adjust: none` in its own shadow DOM styles, then the `::slotted(*)` style might win, but I’m not 100% sure. But any light DOM styles are more specific, so it shouldn't be an issue.